setup.py

```diff
@@ -197,60 +196,24 @@
 """
 Search and set math library environemnt
 """
-_mkl_rpath = []
+_mathlib_rpath = []
 _cmplr_rpath = []
 _omp_rpath = []
 
 
 """
 Get the math library environemnt
 """
-_mkl_include = None
-_mkl_libpath = None
-
-# try to find math library in environment
-if IS_CONDA_BUILD:
-    _conda_root = os.environ.get("PREFIX", None)
-else:
-    _conda_root = os.environ.get("CONDA_PREFIX", None)
-if _conda_root is not None:
-    _mkl_include_find = os.path.join(_conda_root, "include")
-    _mkl_libpath_find = os.path.join(_conda_root, "lib")
-    _required_header = os.path.join(_mkl_include_find, "oneapi", "mkl.hpp")
-    _required_library = os.path.join(_mkl_libpath_find, "libmkl_sycl.so")
-
-    if (os.path.exists(_required_header) and os.path.exists(_required_library)):
-        print(
-            f"Intel DPNP: using $CONDA_PREFIX based math library. include={_mkl_include_find}, libpath={_mkl_libpath_find}")
-        _mkl_include = [_mkl_include_find]
-        _mkl_libpath = [_mkl_libpath_find]
-
-_mkl_root = os.environ.get("MKLROOT", None)
-if ((_mkl_include is None or _mkl_libpath is None) and (_mkl_root is not None)):  # if MKLROOT was specified then use it
-    # TODO change paths and file names for new version
-    # paths and file names are aligned to beta09 at this moment
-    _mkl_include_find = os.path.join(_mkl_root, "include")
-    _mkl_libpath_find = os.path.join(_mkl_root, "lib", "intel64")
-    _required_header = os.path.join(_mkl_include_find, "mkl_blas_sycl.hpp")
-    _required_library = os.path.join(_mkl_libpath_find, "libmkl_sycl.so")
-
-    if (os.path.exists(_required_header) and os.path.exists(_required_library)):
-        print(
-            f"Intel DPNP: using $MKLROOT based math library. include={_mkl_include_find}, libpath={_mkl_libpath_find}")
-        _mkl_include = [_mkl_include_find]
-        _mkl_libpath = [_mkl_libpath_find]
-
-if _mkl_include is None or _mkl_libpath is None:
-    raise EnvironmentError("Intel DPNP: Unable to find math library")
+_mathlib_include, _mathlib_path = find_mathlib(verbose=True)
```

Comment:
Could you please use the same approach for `find_cmlr` (for the code below)?

Reply:
If you like I can merge this PR and next step will be in other PR

Reply:
Let me do the next step in other PR.